src/preferences/colorpalettesettings.cpp

```diff
@@ -1,13 +1,15 @@
 #include "preferences/colorpalettesettings.h"
 
+#include <QRegularExpression>
+
 #include "util/color/predefinedcolorpalettes.h"
 
 namespace {
 const mixxx::RgbColor kColorBlack(0x000000);
 const QString kColorPaletteConfigGroup = QStringLiteral("[Config]");
 const QString kColorPaletteGroupStart = QStringLiteral("[ColorPalette ");
 const QString kColorPaletteGroupEnd = QStringLiteral("]");
-const QRegExp kColorPaletteGroupNameRegex(QStringLiteral("^\\[ColorPalette (.+)\\]$"));
+const QRegularExpression kColorPaletteGroupNameRegex(QStringLiteral("^\\[ColorPalette (.+)\\]$"));
```

Comment:
I recommend to switch to assignment-style initialization as already mentioned. In ancient C++ times this wasn't optimized away, but now it is handled like direct initialization.

Reply:
Don't mind, the constructor is explicit, doesn't make any sense here.